data/static/base_info.csv

```diff
@@ -34,9 +36,11 @@ Luxembourg,,Government,,0
 Mercy Corps Europe,2012-07-02,International NGO,mce,0
 Netherlands - Ministry of Foreign Affairs,2011-09-19,Government,minbuza_nl,54.15
 Netherlands Red Cross,2016-04-29,International NGO,nlrc,44
+New Zealand - Ministry of Foreign Affairs and Trade - New Zealand Aid Programme,2013-06-31,Government,mfat,
```

Comment:
Joined on 24th of October 2017. Baseline value should be recalculated for 1 November 2017. It has been agreed that the first of the month will be taken for any new signatories.